extensions-contrib/time-min-max/src/main/java/io/druid/query/aggregation/TimestampAggregatorFactory.java

```diff
@@ -86,6 +87,12 @@ public Object combine(Object lhs, Object rhs)
   }
 
   @Override
+  public AggregateCombiner makeAggregateCombiner()
+  {
+    throw new UOE("[%s] is not supported during ingestion for rollup", getClass().getSimpleName());
```

Comment:
It doesn't make sense, and useless: only metric columns are aggregated during indexing, not the timestamp column.